plop-templates/Component/stories.tsx.hbs

```diff
@@ -0,0 +1,17 @@
+import { storiesOf } from "@storybook/react";
+import React from "react";
+import { IntlProvider } from "react-intl";
+
+import { MemoryRouter } from "react-router";
+import { {{pascalCase name}} } from ".";
+import { I{{pascalCase name}}Props } from "./{{pascalCase name}}";
+
+const DEFAULT_PROPS: I{{ pascalCase name }}Props = {};
+
+storiesOf("@components/{{ componentType }}/{{pascalCase name}}", module)
+.addParameters({ component: {{pascalCase name}} })
+.addDecorator(story => (
+<IntlProvider locale="en">story()</IntlProvider>
```

Comment:
As `IntlProvider` is always added as decorator, I think it is better to add it just in one place in `.storybook/OutLineDecorator.jsx`, so this decoration addition would be completely removed.

Reply:
Its outside of the scope of this PR, editing existing PRs would be needed. Created Issue: https://app.clickup.com/t/2549495/SALEOR-1194